cli/notifications_test.go

```diff
@@ -16,6 +16,16 @@ import (
 	"github.com/coder/coder/v2/testutil"
 )
 
+func createOpts(t *testing.T) *coderdtest.Options {
```

Comment:
suggestion:

```
func NewWithDatabase(t testing.TB, options *Options, muts ...func(*Options)) { ... }

func WithExperiments(es ...codersdk.Experiment) func(*Options) { ... }

// then:
ownerClient, db := coderdtest.NewWithDatabase(t, nil, coderdtest.WithExperiments(codersdk.ExperimentNotifications))
```

Reply:
Yeah that would be nicer indeed.
I'll defer that to a subsequent PR though if you're ok with that?

Reply:
💯